packages/server/src/ui/routes/build-view/build-hash-selector.css

```diff
@@ -122,7 +121,6 @@
   flex-grow: 1;
   flex-shrink: 10;
   text-overflow: ellipsis;
-  white-space: pre;
```

Comment:
instead of deleting this line, let's just use `nowrap`

Reply:
@patrickhulce tested it with nowrap
![image](https://user-images.githubusercontent.com/232120/133048756-71b43b18-649e-4286-b56f-7ad38c1c2d32.png)
tested it with wrap
![image](https://user-images.githubusercontent.com/232120/133048523-d7712e7c-f49f-4b00-97b6-06b28817d65a.png)

I think the wrap is better but it your call

Reply:
`nowrap` was the intended design here, let's stick with that

Reply:
I changed it to nowrap